main/kotlin/com/example/people_managment_system/service/SystemService.kt

```diff
@@ -0,0 +1,177 @@
+package com.example.people_managment_system.service

```

Comment:
Good stuff on this, now this is what we would call the business logic, operations to perform specific tasks such as getting an employee.

Instead of putting this under the `service` package. I'd recommend changing it to `usecases`.

Similar to the controller, this class is too big, it's doing too much. We need to separate it out into different classes.

Let's use the Employee as the example. We want to perform some actions on it. So, we will create a package called `usecases/employee`. Inside this package we will have a class called `EmployeeService`. Here you can put all the operations related only to Employee.

I also recommend implementing to an interface. Meaning the `EmployeeService` class will have a blueprint of what methods it should implement.

You can read more about it here: https://www.geeksforgeeks.org/implement-interface-in-kotlin/

I will let you figure the other classes out.